### PR TITLE
Remi/components/testimonials branch

### PR DESCRIPTION
**Pull request by Remi-dee**
## This PR closes
- #6 
- #2

### Description 
- Created testimonial component from figma
- Created Navbar component from figma

### View

shadow

### CODE REVIEW

**Review by Oyinbrakemigrace — APPROVED** [review]

**Comment by Remi-dee** [comment]
> ## This PR closes
> * [Create Testimonial Section from Figma Design #6](https://github.com/world-wide-techies/annon_chat/issues/6)
> * [Create Navbar component from Figma design #2](https://github.com/world-wide-techies/annon_chat/issues/2)
> 
> ### Description
> * Created testimonial component from figma
> * Created Navbar component from figma
> 
> ### View
> shadow

Don't review yet

**Review by annonymousauthority — APPROVED** [review]
Seen

File: app/components/Nav_comp.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+function Nav() {
```
Comment (annonymousauthority) [inline app/components/Nav_comp.js]: I'd be more comfortable with a proper elaborate name. Nav is not explanatory enough

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: app/components/Nav_comp.js (hunk 1 above)
Comment (Remi-dee) [reply app/components/Nav_comp.js]: Alright, done with that

**Review by annonymousauthority — APPROVED** [review]
Seen

**Pull request merged**